site/astro.config.js

```diff
@@ -0,0 +1,121 @@
+// @ts-check
+import { defineConfig } from 'astro/config'
+import sitemap from '@astrojs/sitemap'
+import starlight from '@astrojs/starlight'
+
+// https://astro.build/config
+export default defineConfig({
+  site: 'http://cli.netlify.com',
+  trailingSlash: 'ignore',
+  integrations: [
+    sitemap({
+      lastmod: new Date(),
+    }),
+    starlight({
+      credits: true,
+      title: 'Netlify CLI command reference',
+      description: 'Full command reference for the Netlify CLI',
+      social: {
+        github: 'https://github.com/netlify/cli',
+      },
+      favicon: 'favicon.svg',
+      logo: {
+        light: './src/assets/logo-light.svg',
+        dark: './src/assets/logo-dark.png',
```

Comment:
lol I found one inline light SVG somewhere that I turned into a file, and a dark PNG elsewhere... seem weird but it's probably fine?

Reply:
oh, well, actually the dark mode image seems to be wonky, so this needs to be fixed

Reply:
https://developers.netlify.com/_astro/logo-dark.CGIhDU-N.svg?

Reply:
I managed to get one from #crew-brand-design 😀